spec/spec_helper.rb

```diff
@@ -10,6 +10,11 @@
 require 'bundler'
 include RspecPuppetFacts
 
+if ENV['DEBUG']
```

Comment:
this file comes from our modulesync_config repository. we should add it there as well. https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/spec/spec_helper.rb.erb